packages/store/src/lib/Store.ts

```diff
@@ -870,7 +870,8 @@ export class Store<R extends UnknownRecord = UnknownRecord, Props = unknown> {
 
 			updateDepth++
 			if (updateDepth > 100) {
-				throw new Error('Maximum store update depth exceeded, bailing out')
+				console.warn('Maximum store update depth exceeded, bailing out')
+				return
```

Comment:
Why not throw here? It's definitely an error and we should abort any transaction it happens in I think?